### PR TITLE
[formrecognizer] update links

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves https://github.com/Azure/azure-sdk-for-python/issues/20973

### CODE REVIEW

**Review by catalinaperalta — APPROVED** [review]
🚢

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 This version of the SDK defaults to the latest supported API version, which currently is v2021-09-30-preview.
 
-> Note: Starting with version 2021-09-30-preview, a new set of clients were introduced to leverage the newest features of the Form Recognizer service. Please see the Migration Guide for detailed instructions on how to update application code from client library version 3.1.X or lower to the latest version. Also, please refer to the README for more information about the library. 
+> Note: Starting with version 2021-09-30-preview, a new set of clients were introduced to leverage the newest features of the Form Recognizer service. Please see the [Migration Guide][https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md] for detailed instructions on how to update application code from client library version 3.1.X or lower to the latest version. Also, please refer to the [README][https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/formrecognizer/azure-ai-formrecognizer/README.md] for more information about the library.
```
Comment (catalinaperalta) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:7]: WAIT dont merge. These links are broken

Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:7]:
```suggestion
> Note: Starting with version 2021-09-30-preview, a new set of clients were introduced to leverage the newest features of the Form Recognizer service. Please see the [Migration Guide](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md) for detailed instructions on how to update application code from client library version 3.1.X or lower to the latest version. Also, please refer to the [README](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/formrecognizer/azure-ai-formrecognizer/README.md) for more information about the library. 
```

Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:7]: good catch!

**Comment by nrvnujd** [comment]
Hello

pip install azure-ai-formrecognizer --pre is still downloading azure_ai_formrecognizer-3.1.2 package.

Grateful for any suggestions regarding this issue.

**Comment by kristapratico** [comment]
> Hello
> 
> **pip install azure-ai-formrecognizer --pre** is downloading azure_ai_formrecognizer-3.1.2 package.
> 
> Grateful for any suggestions regarding this issue.

Hi @nrvnujd, can you tell which version of azure-ai-formrecognizer you want to install? The 3.1.2 is the latest GA release, if you want to revert to an older, beta version you can specify the version in the command:

`pip install azure-ai-formrecognizer==3.1.0b4`

**Comment by nrvnujd** [comment]
I want to use the 3.2.0b1 - Latest beta release with DocumentAnalysisClient and DocumentModelAdministrationClient. 
As per instructions, **pip install azure-ai-formrecognizer --pre** should be pointing to this version.

**Comment by kristapratico** [comment]
@nrvnujd Ah okay. Version 3.2.0b1 has not been released yet (note that this repo is used for active development of the Python SDK and so what you see here may not be published to PyPi quite yet)

However, version 3.2.0b1 is scheduled to be released this week. I can ping this thread with an update when it is available on PyPi. Thanks for your patience. :)

**Comment by nrvnujd** [comment]
Yes. That would be great. Thank you. 😄

**Pull request merged**